validate.go

```diff
@@ -192,6 +194,48 @@ func checkProcess(process rspec.Process, rootfs string) {
 	}
 }
 
+func supportedMountTypes() (map[string]bool, error) {
+	supportedTypes := make(map[string]bool)
+	f, err := os.Open("/proc/filesystems")
+	if err != nil {
+		return supportedTypes, err
+	}
+	defer f.Close()
+
+	s := bufio.NewScanner(f)
+	for s.Scan() {
+		if err := s.Err(); err != nil {
+			return supportedTypes, err
+		}
+
+		text := s.Text()
+		parts := strings.Split(text, "\t")
+		if len(parts) > 1 {
+			supportedTypes[parts[1]] = true
+		} else {
+			supportedTypes[parts[0]] = true
+		}
+	}
+
+	supportedTypes["nfs"] = true
+	supportedTypes["bind"] = true
```

Comment:
In runtime-spec's mount example, I see `{
        "destination": "/data",
        "type": "bind",
        "source": "/volumes/testing",
        "options": ["rbind","rw"]
    }`, so I think bind may must be supported.